cmd/nfd-topology-updater/main.go

```diff
@@ -109,8 +140,10 @@ func initFlags(flagset *flag.FlagSet) (*topology.Args, *resourcemonitor.Args) {
 		"Time to sleep between CR updates. Non-positive value implies no CR updatation (i.e. infinite sleep). [Default: 60s]")
 	flagset.StringVar(&resourcemonitorArgs.Namespace, "watch-namespace", "*",
 		"Namespace to watch pods (for testing/debugging purpose). Use * for all namespaces.")
-	flagset.StringVar(&resourcemonitorArgs.KubeletConfigFile, "kubelet-config-file", source.VarDir.Path("lib/kubelet/config.yaml"),
-		"Kubelet config file path.")
+	flagset.StringVar(&resourcemonitorArgs.KubeletConfigURL, "kubelet-config-url", "",
```

Comment:
nit: i'd use just `kubelet-config` or then `kubelet-config-uri` which is more accurate